peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -191,6 +191,17 @@ <h1>{{ document.title }}</h1>
                           </dd>
                         {% endif %}
                       {% endblock %}
+                      {% block document-metadata-content-publication %}
```

Comment:
I think this should be near, and be displayed very similarly to, the attached files. So it would be something like:

```
Original publication: Download PDF (32KB)
```

Reply:
See the `document-metadata-media-summary-download` block in document_detail.html